### ISSUE TITLE
jest tests fail

### ISSUE THREAD

**Issue by jhillert-dmg** [benyap/resolve-tspaths#95]
I am not sure if this is a bug, something I overlooked in the documentation or an incorrect tsconfig configuration, but the situation is this.  We have a typescript based project that consumes, via npm, a library of common components that we also maintain.  We added resolve-tspaths to the component library.  The projects that import the component library as a dependency recognize the hooks alias and run fine, but every jest test fails with an error like this (hooks is one of the aliases):

> Cannot find module 'hooks' from 'node_modules/@dmgtech/cc-shared/lib/cjs/components/InputField/styles.js'

ts-config for component library is this:
![image](https://user-images.githubusercontent.com/85570613/172048257-5633caeb-2bc5-47ac-8506-56066fab7c84.png)

package.json scripts used to build component library is this:
![image](https://user-images.githubusercontent.com/85570613/172048304-0023498d-095b-4caa-a168-ccfa86952cab.png)

I assumed the problem was the build:cjs script so I tried this but got "changed 0 files"
![image](https://user-images.githubusercontent.com/85570613/172048347-d642326c-c5ca-49d4-a644-c1be46e760f5.png)

To sum up, and restate the problem, the tests of the projects that use our aliased component library as a dependency are failing, and they are failing because the the aliases in the component library are not getting mapped in the cjs files.

**Comment by jhillert-dmg** [comment]
Turns out I needed to spend more time reading your README file. ` && resolve-tspaths --out lib/cjs` worked.
